ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__acm-ocp4.14-lp-interop.yaml

```
@@ -49,19 +49,19 @@ tests:
-          {"step": "acm-tests-obs", "failure_type": "all", "classification": "OBS testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["CI", "Observability", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
-          {"step": "acm-tests-grc", "failure_type": "all", "classification": "GRC testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["CI", "GRC", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
-          {"step": "acm-tests-alc", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "jira_additional_labels": ["CI", "ApplicationLifecycle", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-mch", "failure_type": "all", "classification": "MCH install", "jira_project": "LPTOCPCI", "group": {"name": "lp-tests", "priority": 1}},
+          {"step": "acm-tests-clc-create", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["CI", "ClusterLifecycle", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-tests-clc-destroy", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["CI", "ClusterLifecycle", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-fetch-*", "failure_type": "all", "classification": "Fetch managed cluster failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["CI", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-tests-obs", "failure_type": "all", "classification": "OBS testcase failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["CI", "Observability", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-tests-grc", "failure_type": "all", "classification": "GRC testcase failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["CI", "GRC", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
+          {"step": "acm-tests-alc", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "ACM", "jira_component": "QE", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["CI", "ApplicationLifecycle", "QE-ACM"], "jira_assignee": "vboulos@redhat.com"},
           {"step": "acm-must-gather", "failure_type": "pod_failure", "classification": "ACM must-gather failure", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}},
```

```suggestion
          {"step": "acm-must-gather", "failure_type": "pod_failure", "classification": "ACM must-gather failure", "jira_project": "LPTOCPCI", "group": {"name": "lp-tests", "priority": 2}},
```

I think we should consider this step a part of the lp-tests group.
The reason I think this is because we just saw this scenario fail in the install-operators step. When that happens it means the acm operator was never installed which means that the acm-must-gather step will automatically fail if the install-operators step fails. 

So marking this as the last priority of the lp-tests group makes the most sense to me. What do you think?